### PR TITLE
Fix cut off slider value

### LINKED ISSUE #10188: Input Number state not visible in preview (slider cut-off)
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Adding an input_number helper in a card doesn't show the state while picking up a number (in the preview circle). Either alone in a card or with other entities in an entities card the number is obscured 
image

### Describe the behavior you expected

I'd expect the number not to be obscured and to show up as it did in the past within the blue circle.

### Steps to reproduce the issue

1. Make an input_number helper
2. Add the helper in an entities card
3. Try to change the value from the slider
4.  Be disappointed

### What version of Home Assistant Core has the issue?

core-2021.10.0

### What was the last working version of Home Assistant Core?

core-2021.9.7 (I think)

### In which browser are you experiencing the issue with?

Google Chrome 94.0.4606.71 / Firefox 91.0.2

### Which operating system are you using to run this browser?

Windows 10 / Android

### PR DESCRIPTION
**Pull request by chriss158**
## Proposed change

This PR fixes the cut of slider value.
Screenshot here: #10188

With PR #9837 the input number entity row uses now `display: block`.
With this change the overflow setting (overflow: hidden) of the parent containers is active. This causes the cut off the value of the input slider.

By removing the overflow setting, the value of the input slider does not cut off. Seems for me the simpliest fix, as i don't see a reason to hide the overflow. But maybe i'm wrong here. At least i didn't see any side effects.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #10188, #10192
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by spacegaier** [comment]
There is also another older issue, that is related: https://github.com/home-assistant/frontend/pull/10250

**Comment by Raul-7-7** [comment]
@chriss158 did you have any luck? I tried a few tricks but as you suggested, mixing the overflow values dont seem to work. 
Do you know what is the reason to prevent overflow on x?

**Comment by chriss158** [comment]
> @chriss158 did you have any luck? I tried a few tricks but as you suggested, mixing the overflow values dont seem to work. 
> Do you know what is the reason to prevent overflow on x?

No, i tried also some suggestion i found. But nothing worked. I would allow both sides to overflow as long there is no simple solution to hide the x overflow.

**Comment by jurt** [comment]
`#states > div > * { 
   overflow: clip visible; 
 }
 `
Looks like this variation does the trick, that allows to content overflow vertically while still clipped horizontally. Not sure however if it not breaks some case when vertical scrollbar is required in some #states > div > * element

**Comment by Raul-7-7** [comment]
> `#states > div > * { overflow: clip visible; } ` Looks like this variation does the trick, that allows to content overflow vertically while still clipped horizontally. Not sure however if it not breaks some case when horizontal scrollbar is required in some #states > div > * element

hmm this is interesting, good point about horizontal scrollbar, however, would you not have that problem when you limit on x anyway? so I think your solution might do the trick. 
@chriss158 what do you think?

**Comment by jurt** [comment]
> hmm this is interesting, good point about horizontal scrollbar, however, would you not have that problem when you limit on x anyway? 

As far as I understood the [doc](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#values) difference is only that with _overflow: hidden_ content can still be scrolled programmatically which is not the case with _overflow: clip_
IMO should be acceptable in this particular case

**Comment by chriss158** [comment]
Thanks @jurt and @Raul-7-7. That does indeed work. Ani see it the same way as @jurt. clip should be acceptable.

File: src/panels/lovelace/cards/hui-entities-card.ts [hunk 1]
```diff
@@ -231,7 +231,6 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
         display: flex;
         flex-direction: column;
         justify-content: space-between;
-        overflow: hidden;
```
[the remark this answers is not in the document]
Comment (chriss158) [reply src/panels/lovelace/cards/hui-entities-card.ts:234]: This one doesn't work with the workaround to hide overflow-x with clip that @jurt suggested. Any ideas?

**Review by bramkragten — APPROVED** [review]
I think this will work, let's try, and if not we will look at it again :-)

**Pull request merged**